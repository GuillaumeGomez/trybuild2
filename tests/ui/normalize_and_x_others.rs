fn main() {
    let _x = 42_u8 >> "bar";
}
