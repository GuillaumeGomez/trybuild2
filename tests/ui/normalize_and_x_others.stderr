error[E0277]: no implementation for `u8 >> &str`
 --> tests/ui/normalize_and_x_others.rs:2:20
  |
2 |     let _x = 42_u8 >> "bar";
  |                    ^^ no implementation for `u8 >> &str`
  |
  = help: the trait `Shr<&str>` is not implemented for `u8`
  = help: the following other types implement trait `Shr<Rhs>`:
            <&'a i128 as Shr<i128>>
            <&'a i128 as Shr<i16>>
            <&'a i128 as Shr<i32>>
            <&'a i128 as Shr<i64>>
            <&'a i128 as Shr<i8>>
            <&'a i128 as Shr<isize>>
            <&'a i128 as Shr<u128>>
            <&'a i128 as Shr<u16>>
          and $COUNT others
